components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesMvcEndpoint.java

```diff
@@ -60,7 +60,7 @@ public CamelRoutesMvcEndpoint(CamelRoutesEndpoint delegate) {
 
     @ResponseBody
     @GetMapping(
-        value = "/{id}/info",
+        value = {"/{id}/info", "/{id}/detail"},
```

Comment:
I think we should not have two paths serving the same purpose so I'd go for detail only